### ISSUE TITLE
DomMarker does not work in Shadow DOM

### ISSUE THREAD

**Issue by hellokatili** [heremaps/maps-api-for-javascript-examples#105]
Using `DomMarker` with `DomIcon` within a shadow DOM creates the following error on every mouse movement in the map:
```
mapsjs.bundle.js:4 Uncaught TypeError: Cannot read property 'getPropertyValue' of undefined
    at mm.Ha (mapsjs.bundle.js:4)
    at Xm (mapsjs.bundle.js:4)
    at S.Ha (mapsjs.bundle.js:4)
    at T.$d (mapsjs.bundle.js:4)
    at Ho (mapsjs.bundle.js:4)
    at Go (mapsjs.bundle.js:4)
    at No.g (mapsjs.bundle.js:4)
```

**Comment by dargains** [comment]
Hello, I'm also having this issue when using Heremaps inside a Shadow DOM. 
@hellokatili did you manage to solve this issue?
Thanks!

**Comment by dbacarel** [comment]
The bug related to mouse events on DomMarker within shadow DOM was fixed since v3.1.22.1. Please reopen this issue if you still face this issue.
